platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationStore.java

```diff
@@ -33,7 +33,8 @@
 
   private Double nearestNeighborDistanceLimit;
 
-  private ConfigurationStore() {}
+  public ConfigurationStore() { // nothing to do
```

Comment:
❓Can we avoid having a `public` constructor _and_ a `getInstance()` method? It makes it confusing to know which one to use when a `ConfigurationStore` object is needed. Should clients create a new one or use the singleton?

Reply:
I updated the doc and created the following ticket to track removing the singleton from this class: https://codice.atlassian.net/browse/DDF-3781
👍